common/lib/protocol-definitions/tsconfig.json

```diff
@@ -6,9 +6,10 @@
     ],
     "compilerOptions": {
         "rootDir": "./src",
-        "outDir": "./dist"
+        "outDir": "./dist",
+        "types": ["node"]
```

Comment:
adding a dependency on node here is probably not something we want. we've actually done a bunch of work to keep our code isomorphic, and reduce the need to node polyfills on the client. I worry that re-adding node types could result in the spread of node specific functionality again